src/main/java/uk/gov/hmcts/cmc/claimstore/services/OffersService.java

```diff
@@ -97,7 +99,7 @@ public Claim countersign(Claim claim, MadeBy party, String authorisation) {
         settlement.countersign(party);
 
         caseRepository.reachSettlementAgreement(claim, settlement, authorisation, SETTLED_PRE_JUDGMENT.name());
-        Claim updated = claimService.getClaimByExternalId(claim.getExternalId(), authorisation);
+        Claim updated = claimService.getClaimByExternalId(claim.getExternalId(), authorisation, CLOSED);
```

Comment:
Have you checked with BA whether closed class should be visible to users dashboards? I cannot recall any requirement to remove settled cases from dashboard.

This seems to be hiding problem, not fixing it.

Reply:
Do we actually close cases at settlement? I didn't think we had implemented that

Reply:
<img width="1387" alt="screen shot 2018-06-15 at 11 29 44" src="https://user-images.githubusercontent.com/1724691/41463740-7984e834-708f-11e8-9a20-e21df6b9b90a.png">

Reply:
it was never a requirement to not show closed state case following claim store model.